educative_scraper.py

```diff
@@ -165,7 +171,7 @@ def create_html(file_name, base_64_png, html_template):
                 <head>
                     <title>{file_name}</title>
                 </head>
-                <body style="zoom: 80%">
+                <body style="zoom: 100%">
```

Comment:
reset this to 80%, zoom 100% causes image distortion and reduces quality.
it also affects educative-viewer

Reply:
done, Zoom set back to 80%.